runtime/compiler/trj9/optimizer/JProfilingValue.cpp

```diff
@@ -228,7 +228,7 @@ TR_JProfilingValue::removeRedundantProfilingValueCalls()
                node->getFirstChild()->getThirdChild()->getReferenceCount() == 1,
                "Second and Third Child of the value calls should be referenced only once");
             */
-            dumpOptDetails(comp(), "%s Removing profiling treetop, node n%dn is already profiled before\n",
+            dumpOptDetails(comp(), "%s Removing profiling treetop, node n%dn is already profiled\n",
```

Comment:
this should be a perform transformation since it isn't wrong to profile the same value multiple times.